### PR TITLE
schedule: fix the zero store limit

### LINKED ISSUE #2566: Panic when upgrading PD to the latest master
### What did you do?

Upgrade PD to the latest master.

### What did you expect to see?

successfully upgrade

### What did you see instead?
```
[FATAL][log.go:292][panic][recover: "\"cannot find suitable quantum for 0\""]
```

### What version of PD are you using (`pd-server -V`)?
master

### PR DESCRIPTION
**Pull request by rleungx**
### What problem does this PR solve?
Fix #2566.

### What is changed and how it works?

This PR will init a default store limit config for stores when upgrading from the previous version.

### Check List

Tests

- Unit test

Related changes

- Need to cherry-pick to the release branch

### Release note

- Fix the issue that PD may panic when upgrading from the previous version

### CODE REVIEW

File: server/config/persist_options.go [hunk 1]
```diff
@@ -270,7 +270,14 @@ func (o *PersistOptions) GetStoreLimit(storeID uint64) StoreLimitConfig {
 	if limit, ok := o.GetScheduleConfig().StoreLimit[storeID]; ok {
 		return limit
 	}
-	return StoreLimitConfig{0, 0}
+	cfg := o.GetScheduleConfig().Clone()
```
Comment (nolouch) [inline server/config/persist_options.go:273]: Could you add a unit test?

Comment (rleungx) [reply server/config/persist_options.go:273]: Done

File: server/config/persist_options.go [hunk 2]
```diff
@@ -270,7 +270,14 @@ func (o *PersistOptions) GetStoreLimit(storeID uint64) StoreLimitConfig {
...
+	sc := StoreLimitConfig{
+		AddPeer:    DefaultStoreLimit.GetDefaultStoreLimit(storelimit.AddPeer),
+		RemovePeer: DefaultStoreLimit.GetDefaultStoreLimit(storelimit.RemovePeer),
```
Comment (HunDunDM) [inline server/config/persist_options.go:276]: It could also be a TiFlash node?

**Pull request merged**